SConstruct

```diff
@@ -929,7 +929,8 @@ elif env["OS"] == "Darwin":
     config.select("macOS")
 
 # SHLIBVERSION fails with MinGW: http://scons.tigris.org/issues/show_bug.cgi?id=3035
-if (env["toolchain"] == "mingw"):
+# I don't think this does anything? env["toolchain"] seems to be a list of strings, not a single string.
+if env["toolchain"] == "mingw":
     config.select("mingw")
```

Comment:
@speth @ischoegl Question here, is this line still relevant? As I wrote in the comment, this line doesn't seem to ever be true. Do we need it?

Reply:
Not sure ... you could test on the mingw runner on GH though.

Reply:
Yeah, I know we set the toolchain to include mingw. The question is more like, `env["toolchain"]` is always a list of tools, so the `==` comparison will always be false as far as I can tell. So line 934 never gets called and things seem fine :shrug:

Reply:
It’s probably a bug then … the `config.select` is supposed to do something if I recall correctly.